src/scala/microsoft-spark-3-1/src/main/scala/org/apache/spark/api/dotnet/CodegenUtils.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the .NET Foundation under one or more agreements.
+ * The .NET Foundation licenses this file to you under the MIT license.
+ * See the LICENSE file in the project root for more information.
+ */
+
+package org.apache.spark.api.dotnet
+
+import org.apache.spark.ml._
+import scala.collection.JavaConverters._
+
+object DotnetHelper {
```

Comment:
Should we name these as `MLDotnetHelper` perhaps? Or will they be generic? And can we add a short description of this object and its purpose?

Reply:
Except the first `setPipelineStages` method is specific to ML's Pipeline, the other two methods could be generic that can be used as general java-scala data type conversion methods.